packages/jsapi-bootstrap/src/useWidget.ts

```diff
@@ -97,7 +109,17 @@ export function useWidget<T extends WidgetTypes = dh.Widget>(
     [descriptor, objectFetch]
   );
 
-  return wrapper;
+  const wrapperWithApi = useMemo(
+    () => ({
+      ...wrapper,
+      error:
+        wrapper.error ?? (descriptorApiError as NonNullable<unknown> | null),
```

Comment:
Maybe move this to `useDeferredApi` where the error is defined there? Would be a little more correct I think, either the error is a non nullable unknown or it's null.